### ISSUE TITLE
zero_grad() 질문 있습니다.

### ISSUE THREAD

**Issue by High-East** [heartcored98/Standalone-DeepLearning#11]
epoch 당 zero_grad()를 한 번 사용하시던데요!
loss.backward()를 사용할 때마다 그 전에 zero_grad()를 해야 하는게 아닌가요??

**Comment by heartcored98** [comment]
앗 그런 실수를 했었군요..!  혹시 어떤 Lecture 였는지 링크를 알려주시겠어요?

**Comment by High-East** [comment]
https://github.com/heartcored98/Standalone-DeepLearning/blob/master/Lec5/Lab7_CIFAR-10_with_CNN.ipynb

**Comment by heartcored98** [comment]
@High-East 회사 일로 정신이 없었다가 이제 반영하였습니다. 여러 랩 코드에 있었던 동일한 오류를 모두 수정하였습니다..! 더 일찍 수정하지 못해 죄송하며 앞으로도 문제가 있다면 언제든지 알려주세요..!
